tests/unit/stores/pacbioRuns.spec.js

```diff
@@ -0,0 +1,102 @@
+import { setActivePinia, createPinia } from 'pinia'
```

Comment:
Would it be worth adding this to the testHelper and importing? The same as we have done with Vuex.